src/hardware/input/intel8042.cpp

```diff
@@ -46,16 +46,16 @@ CHECK_NARROWING();
 // - http://www.os2museum.com/wp/ibm-pcat-8042-keyboard-controller-commands/
 // - http://www.os2museum.com/wp/ibm-ps2-model-50-keyboard-controller/
 
-static constexpr uint8_t irq_num_kbd_pcjr  = 6;
-static constexpr uint8_t irq_num_kbd_ibmpc = 1;
-static constexpr uint8_t irq_num_mouse     = 12;
+static constexpr uint8_t IrqNumKbdPCJr  = 6;
```

Comment:
Nitpicking: Pcjr

Reply:
If you remember, we had similar discussions before and decided not to mimic "marketing capitalisation", and thus just went with `VmWare`.

Reply:
Ok, changed.